ansible/roles/tester/tasks/main.yml

```diff
@@ -25,3 +25,10 @@
     name: VMWare.PowerCLI
     state: present
 
+- name: Check TrustedHosts value
+  win_shell: (Get-Item WSMan:\localhost\Client\TrustedHosts).Value
+  register: trusted_hosts
+
+- name: Set Testbeds as TrustedHosts
+  win_shell: "Set-Item WSMan:\\localhost\\Client\\TrustedHosts -Value \"{{ (((trusted_hosts.stdout | trim()).split(',') | reject('equalto', '') | list()) + ['172.17.0.*']) | join(',') }}\" -Force"
```

Comment:
You can use the [`>` style syntax](http://docs.ansible.com/ansible/latest/YAMLSyntax.html) next time, so you don't have to do all these escapes. I think it would look like:

```yaml
  win_shell: >
    Set-Item WSMan:\localhost\Client\TrustedHosts
    -Value "{{ (((trusted_hosts.stdout | trim()).split(',') | reject('equalto', '') | list()) + ['172.17.0.*']) | join(',') }}"
    -Force
```

Also, https://stackoverflow.com/a/21699210